src/base/emu-i386/kvm.c

```diff
@@ -153,6 +153,9 @@ static int kvmfd, vmfd, vcpufd;
 static struct kvm_sregs sregs;
 
 #define MAXSLOT 400
+/* flags bits 16-31 will not be used by the KVM API so we can use
+   them internally */
```

Comment:
Is that documented in kvm api?

Reply:
it's in the uapi header linux/kvm.h
```
/*
 * The bit 0 ~ bit 15 of kvm_memory_region::flags are visible for userspace,
 * other bits are reserved for kvm internal use which are defined in
 * include/linux/kvm_host.h.
 */
```
but I think memcheck_is_system_ram() from #1972 can replace it.